### PR TITLE
chore(deps): update jest

### PR DESCRIPTION
**Pull request by rabelloo**
## Purpose

Update Jest related dependencies.

## Approach

`npx npm-upgrade` then `yarn`

## Testing

CI

## Risks

None.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-14
+14.17
```
Comment (josokinas) [inline .nvmrc:1]: That's interesting, so Jest requires a minimum specific minor version of Node? 🤔

Also, worth updating `CONTRIBUTING.md`, it has also mention of Node 14 version.

**Review by josokinas — APPROVED** [review]
Looks good, so it wasn't picked up by Dependabot?

**Comment by rabelloo** [comment]
> Looks good, so it wasn't picked up by Dependabot?

It was, but it couldn't automatically resolve the Node version bump.

File: .nvmrc (hunk 1 above)
Comment (rabelloo) [reply .nvmrc:1]: Apparently so, or at least lists it as a dependency and `yarn` wants to enforce it.

Will update that file, thanks for the reminder.

**Pull request merged**